Neo4j.Driver/Neo4j.Driver/V1/Types/IValue.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) 2002-2018 "Neo Technology,"
+// Network Engine for Objects in Lund AB [http://neotechnology.com]
+// 
+// This file is part of Neo4j.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+namespace Neo4j.Driver.V1
+{
+    public interface IValue
```

Comment:
Seems only Point is a `IValue` now. I assume `IValue` type is only for Cypher supported Special Parameter types.
Pls add some docs for this new public API

Reply:
I do not know how to write this comment to class `ValueExtensions.cs`, so I am putting it here.

About `ValueExtensions.cs`, can you move the path of `ValueExtensions.cs`? it is in the public API. Will this enforce everyone to change the import path of this class?